a.rb

```diff
@@ -0,0 +1,5 @@
+p "hoge"
+
+def foo val
+  p "piyo"
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Prefer single-quoted strings when you don't need string interpolation or special symbols.